c2rust-transpile/src/translator/literals.rs

```diff
@@ -116,6 +116,13 @@ impl<'c> Translation<'c> {
                 };
                 let val = match self.ast_context.resolve_type(ty.ctype).kind {
                     CTypeKind::LongDouble => {
+                        if ctx.is_const {
+                            return Err(format_translation_err!(
```

Comment:
Do these errors bubble up and indicate that we can't translate this as a const macro? Or does the transpiler just error?